bfsfs/bfsfs.go

```diff
@@ -29,15 +29,17 @@ import (
 	"io/ioutil"
 	"net/url"
 	"os"
+	"path"
 	"path/filepath"
 
 	"github.com/bsm/bfs"
 )
 
 func init() {
 	bfs.Register("file", func(_ context.Context, u *url.URL) (bfs.Bucket, error) {
```

Comment:
it would be nice to have a test which tests Resolve with `"file://testdata/root"` and `"file://" + filepath.Abs("./testdata/root")`  - add a `bfsfs/testdata/root/file.txt`

Reply:
@dim what about unit test for this?

> I can (should?) try to make a properly tested, IDK... bfsfs/internal/pathtool.BuildRoot(u *url.URL) string or so.

Because testing Resolve is a bit trickier (and will cover only `Head` basically (I'd choose `Head` to simplify the test)).

Reply:
nah, don't bother with making this too complicated. it was more like resolve the ^^ above, do a Head on 'file.txt', done!

Reply:
I'll go with unit test for now.